<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramEqualizationOptionWidget</class>
 <widget class="QWidget" name="HistogramEqualizationOptionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>93</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="redCheckBox">
     <property name="text">
      <string>Red</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="greenCheckBox">
     <property name="text">
      <string>Green</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="blueCheckBox">
     <property name="text">
      <string>Blue</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmButton">
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
